src/OrchardCore.Modules/OrchardCore.Autoroute/Drivers/AutoroutePartDisplay.cs

```diff
@@ -110,12 +112,29 @@ public override async Task<IDisplayResult> UpdateAsync(AutoroutePart model, IUpd
                     await updater.TryUpdateModelAsync(model, Prefix, t => t.SetHomepage);
                 }
 
-                var errors = await context.ValidateAsync(model);
-                updater.ModelState.BindValidationResults(Prefix, errors);
+                updater.ModelState.BindValidationResults(Prefix, model.ValidatePathFieldValue(S));
+
+                // This can only validate the path if the Autoroute is not managing content item routes or the path is absolute.
+                if (!String.IsNullOrEmpty(model.Path) && (!settings.ManageContainedItemRoutes || (settings.ManageContainedItemRoutes && model.Absolute)))
```

Comment:
Yeah so this is why we were always undecided about sharing the validation code.

The logic is similar between the handler and here (maybe even the same) but is used in different ways.